prow/git/git.go

```diff
@@ -57,6 +57,8 @@ type Client struct {
 	// base is the base path for git clone calls. For users it will be set to
 	// GitHub, but for tests set it to a directory with git repos.
 	base string
+	// host is the git host.
+	host string
```

Comment:
I don't understand the distinction between `base` and `host` -- why do we need both?

Reply:
We don't. I made it like this, only because I want to make minimal changes in this file as you requested earlier, and somehow I agree because you're proposing some big changes here, too. I don't want to bother you if you need to revisit here to refresh memory when it comes to rebase.

Reply:
Understood -- could you add a quick comment here to explain

Reply:
Yeah, sure.